pkg/util/ec2/network_test.go

```diff
@@ -23,11 +23,16 @@ func TestGetPublicIPv4(t *testing.T) {
 	ip := "10.0.0.2"
 	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Content-Type", "text/plain")
-		switch r.RequestURI {
-		case "/public-ipv4":
-			io.WriteString(w, ip)
-		default:
-			w.WriteHeader(http.StatusNotFound)
+		switch r.Method {
+		case http.MethodPut: // token request
+			io.WriteString(w, "AQAAAFKw7LyqwVmmBMkqXHpDBuDWw2GnfGswTHi2yiIOGvzD7OMaWw==")
```

Comment:
nit: can you add a comment explaining where you got this/confirming that this is in fact safe to check in

Reply:
I copied this from another test :) I will refactor this to a constant so it's clearer. I'm guessing it's a fake token, these tests don't need a real one, just any string (since the IMDS server is local as well and doesn't do auth)